lib/types.ts

```diff
@@ -32,6 +32,12 @@ export interface PluginMetadata {
   imageLayers: string[];
 }
 
+interface EcosystemHints {
```

Comment:
I think an export is missing here

Reply:
it's not used anywhere in the code, but let's please focus more on the message (the name and the placement in the data structure) and less on the implementation detail :) it's a draft